content/e20co/drone-marathon/index.md

```diff
@@ -1,11 +1,11 @@
 ---
-author: "dellintosh"
 date: 2017-01-06T18:00:00-06:00
-image: "e20co/drone-marathon"
-repo: "e20co/drone-marathon"
+title: Marathon
+author: dellintosh
 tags: [ publish, marathon ]
+repo: expansioncap/drone-marathon
```

Comment:
So there are actually a few repos, and I am not sure which one is most up-to-date:

https://github.com/dellintosh/drone-marathon
https://github.com/expansioncap/drone-marathon
https://github.com/drone-plugins/drone-marathon